protocols/identify/src/protocol.rs

```diff
@@ -53,6 +53,8 @@ pub struct Info {
     pub protocols: Vec<StreamProtocol>,
     /// Address observed by or for the remote.
     pub observed_addr: Multiaddr,
+    /// Verifiable addresses of the peer.
+    pub signed_peer_record: Option<SignedEnvelope>,
```

Comment:
since this is adding a new field to the struct, it breaks semver unless we bump the minor version.